src/sentry/sentry_apps/token_exchange/grant_exchanger.py

```diff
@@ -29,17 +30,31 @@ class GrantExchanger:
     client_id: str
     user: User
 
+    @sentry_sdk.trace
     def run(self):
         with transaction.atomic(using=router.db_for_write(ApiToken)):
+            sentry_sdk.set_context(
+                "token-exchange.grant",
+                {
+                    "user_id": self.user.id,
+                    "install_id": self.install.id,
+                    "client_id": self.client_id,
+                    "org_id": self.install.organization_id,
+                    "sentry_app_id": self.sentry_app.id,
+                    "sentry_app_slug": self.sentry_app.slug,
+                    "application_id": self.application.id,
+                    "grant_id": self.grant.id,
```

Comment:
are Grants sensitive info ?

Reply:
Grant ids aren't sensitive. However like the other logging message you have some redundant data in here that we could omit.